tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py

```diff
@@ -11,6 +11,7 @@
 class Messages(Base):
 
     name = 'Messages'
+    manifest_url = "app://sms.gaiamobile.org/manifest.webapp"
```

Comment:
I thought @jlorenzo said he wanted to use these properties instead of app and gaiamobile.org, because then it was stored in one central place.
By doing this, you're not storing in one central place anymore.
But I'll let jlorenzo decide.